examples/storage_volume_template.go

```diff
@@ -184,10 +185,10 @@ func main() {
 		TemplateProperties: &Properties,
 		Name:               name_to_create,
 		Description:        "Volume template Example",
-		RootTemplateUri:    "/rest/storage-volume-templates/727d5e9a-9ca5-4788-8eb1-ac30011285e7",
+		RootTemplateUri:    "/rest/storage-volume-templates/6ae24a18-59a5-498b-9426-ac6200baa9e6",
```

Comment:
Get root template

Reply:
can you share the endpoint for getting root template?

Reply:
Add functionality in library files. Filter by field where isRoot is true

Reply:
done

Reply:
done